eng/helix/content/runtests.sh

```diff
@@ -30,6 +29,9 @@ RED="\033[0;31m"
 YELLOW="\033[0;33m"
 MAGENTA="\033[0;95m"
 
+echo "cp -R $HELIX_CORRELATION_PAYLOAD/dotnet $DOTNET_ROOT"
+cp -R $HELIX_CORRELATION_PAYLOAD/dotnet $DOTNET_ROOT
```

Comment:
Two points here: 
- Why copy from the correlation payload instead of just running from there?
- Not in the scope of this change but this script looks like it could be improved using $HELIX_WORKITEM_ROOT to get the working directory.

Reply:
I tried installing the runtime to the work item root, and leaving the SDK in the payload, but the SDK wouldn't be found by the runtime when I split them up.  Ideally the helix sdk would enable installing a specific version of the runtime and sdk at the same time.  Yeah I'll clean up the DIR stuff to just use the workitem root variable

Reply:
What do you mean by "SDK wouldn't be found by the runtime"? Why runtime need to find SDK? (SDK has dependency on runtime, not the other way around)

If you set `DOTNET_ROOT`, the exe produced by dotnet build will use that runtime instead